compiler/ir/backend.jvm/codegen/src/org/jetbrains/kotlin/backend/jvm/codegen/irCodegenUtils.kt

```diff
@@ -320,3 +317,27 @@ private fun IrFunction.isAccessorForDeprecatedJvmStaticProperty(context: JvmBack
     val property = callee.correspondingPropertySymbol?.owner ?: return false
     return property.isDeprecatedCallable(context)
 }
+
+val IrClass.getReifiedTypeParameters: ReifiedTypeParametersUsages
```

Comment:
Please, stick to official code style - `getReifiedTypeParameters` is a function, if you use a property, use `reifiedTypeParameters`.

Reply:
Thanks for you suggestion, I have modified it~